Best Practices Checklists/Outdated_Statistics.sql

```diff
@@ -4,6 +4,7 @@ DECLARE
  , @MinimumDaysOld	INT = 35 -- adjust as needed
  , @MaxDOP		INT = NULL -- set to 1 to reduce server workload
  , @SampleRatePercent	INT = NULL -- set to number between 1 and 100 to force a specific sample rate, where 100 = FULLSCAN
+ , @ExcludeSysStats	BIT = 1 --set to 1 to exclude the system stats like '_WA_Sys_%'
```

Comment:
That's a good addition. But can you please set it to 0 as a default value?

Reply:
fixed this as per ur comment